tests/e2e/test_small_file_workload.py

```diff
@@ -0,0 +1,82 @@
+"""
+Test to exercise Small File Workload
+"""
+import logging
+import pytest
+import time
+
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.utility.utils import TimeoutSampler
+from ocs_ci.ocs.resources.ocs import OCS
+from ocs_ci.utility import templating
+from ocs_ci.ocs.utils import get_pod_name
+from ocs_ci.ocs.ripsaw import RipSaw
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import E2ETest, tier1
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='function')
+def ripsaw(request, storageclass_factory):
+    # Create Ceph Block Pool backed PVC
+    storageclass_factory(sc_name='ceph-backed')
+    # Create RipSaw Operator
+    ripsaw = RipSaw()
+
+    def teardown():
+        ripsaw.cleanup()
+    request.addfinalizer(teardown)
```

Comment:
please call this line first

Reply:
I didn't get, teardown will not be defined in that case?

Reply:
it will but if one of the calls:
storageclass_factory(sc_name='ceph-backed')
Or
ripsaw = RipSaw()

the teardown will not be loaded so it won't be executed.
Consider chaning to:
```
@pytest.fixture(scope='function')
￼ def ripsaw(request, storageclass_factory):
    ￼ def teardown():
￼         ripsaw.cleanup()
￼     request.addfinalizer(teardown)

￼     # Create Ceph Block Pool backed PVC
￼     storageclass_factory(sc_name='ceph-backed')
￼     # Create RipSaw Operator
￼     ripsaw = RipSaw()
￼
￼     def teardown():
￼         ripsaw.cleanup()
￼     request.addfinalizer(teardown)
```